docs/2.guide/1.concepts/8.typescript.md

```diff
@@ -12,19 +12,19 @@ To enable type-checking at build or development time, install `vue-tsc` and `typ
 ::code-group
 
   ```bash [yarn]
-  yarn add --dev vue-tsc@^1 typescript
+  yarn add --dev vue-tsc@^2 typescript
   ```
 
   ```bash [npm]
-  npm install --save-dev vue-tsc@^1 typescript
+  npm install --save-dev vue-tsc@^2 typescript
```

Comment:
We can probably remove the major version now.

Reply:
Sure, I removed the version in [a20a3bb](https://github.com/nuxt/nuxt/pull/28484/commits/a20a3bb64f335c1367d4f735e9e9c5a3a758fabb)!